api/librats_collect_evidence.c

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2022 Intel Corporation
```

Comment:
Thank you! I have two suggestions.
1. Could you please provide the API doc about "API of librats", so that user can get started soon
2. Right now, three APIs are provided. API about `get_tcb_status` also should be provided for attestation service.
3. Could you please provide a sample program to show how to use the APIs to complete collecting and verifying evidence?

Reply:
adding get_tcb_status will be stage 2. It will come soon. 
grpc will use librats as attester and verifier and it will publish soon.